test/common/testlib.py

```diff
@@ -784,9 +788,6 @@ def skipImage(reason, *args):
     return lambda func: func
 
 class Policy(object):
-    def __init__(self):
-        self.retryable = True
```

Comment:
The Policy object should not have state, imo.  Now check_retry gets all state passed in that it needs.